lib/ProductOpener/Display.pm

```diff
@@ -8636,6 +8636,15 @@ sub display_nutriscore_calculation_details($) {
 
 	my $nutriscore_data_ref = shift;
 	
+	my $is_beverage;
```

Comment:
Would rename to "beverage_view" or something like this. The is_ prefix makes me think of a bool.